templates/coredns/config.yml

```diff
@@ -0,0 +1,7 @@
+name: Core DNS
+description: |
+  CoreDNS is a DNS server that chains plugins and provides such DNS Services like Etcd & Kubernetes.For kubernetes, please use helm coredns's chart.
+version: 0.8.0
+category: External DNS
+labels:
+  io.rancher.orchestration.supported: 'cattle,mesos,swarm,kubernetes'
```

Comment:
only cattle here？

Reply:
@niusmallnan only cattle, because for kubernetes it already had coredns helm chart, which better support for kubernetes.